src/main/java/com/timeplus/kafkaconnect/TimeplusSinkTask.java

```diff
@@ -60,15 +64,15 @@ public String version() {
     public void start(Map<String, String> props) {
         TimeplusSinkConnectorConfig connectorConfig = new TimeplusSinkConnectorConfig(props);
         this.address = connectorConfig.getAddress();
-        this.apiKey = connectorConfig.getAPIKey();
+        this.apiKey = connectorConfig.getAPIKey().value();
         this.workspace = connectorConfig.getWorkspace();
         this.stream = connectorConfig.getStream();
         this.dataFormat = connectorConfig.getDataformat();
         this.createStream = connectorConfig.getCreateStream();
 
-        this.streamUrl = address + "/" + workspace + "/api/v1beta1/streams/";
-        this.ingestUrl = address + "/" + workspace + "/api/v1beta1/streams/" + stream + "/ingest";
-        this.inferUrl = address + "/" + workspace + "/api/v1beta1/source/infer";
+        this.streamUrl = address + "/" + workspace + TIMEPLUS_API_VERSION_PREFIX + "/streams/";
```

Comment:
nit: as all URLs are generated and stored in the object, I wonder why `address`, `workspace` etc. need to be stored in the object as well ( I didn't read the complete code, I could miss something ).

Reply:
the workspace and url are from configurations